### DISCUSSION TITLE
Feature Request: Hide silenced alerts in alert list visualization

### DISCUSSION THREAD

**Discussion by sdaus** [grafana/grafana#65449] [Dashboards, Panels & Visualizations]
I would like to request a feature that allows me to hide silenced alerts in the alert list visualization. Currently, there is no option to hide silenced alerts, which makes it difficult to quickly identify active alerts.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). 

The issue for this discussion is #43855. Feel free to continue the discussion around this there. Thank you!
